### DISCUSSION TITLE
Working with existing Django project

### DISCUSSION THREAD

**Discussion by scheung38** [fastapi/fastapi#6638] [Questions]
Can I use FASTAPI within an existing Django project? If doing so will it improve some of the latency of the Django endpoint queries? and since it already has /swagger endpoint, /docs from FASTAPI should not interfere with it?

Much appreciated

**Comment by ArcLightSlavik** [comment]
>Can I use FASTAPI within an existing Django project?

Yes
https://fastapi.tiangolo.com/advanced/sub-applications/
https://stackoverflow.com/questions/63726203/is-it-possible-to-use-fastapi-with-django

>If doing so will it improve some of the latency of the Django endpoint queries?

Unlikely

>and since it already has /swagger endpoint, /docs from FASTAPI should not interfere with it?

It should not, if it does u can change where the endpoint points: https://fastapi.tiangolo.com/tutorial/metadata/#openapi-url

**Comment by scheung38** [comment]
But Django does not support async so FastAPI will prevent timeout as current Django sees this as it is blocking call

**Comment by raphaelauv** [comment]
since 3.1 yes -> https://docs.djangoproject.com/en/3.2/topics/async/

this place is for fastAPI issues , go to https://github.com/tiangolo/fastapi/discussions or https://gitter.im/tiangolo/fastapi

could you close your issue , thanks

**Comment by scheung38** [comment]
But it is related to see if using FastAPI can alleviate CSV timeout in Django, if not then FASTAPI won’t help then?

**Comment by ArcLightSlavik** [comment]
@raphaelauv Probably shouldn't send gitter anymore, we switched to discord a while back

**Comment by tiangolo** [comment]
Thanks for the help here @raphaelauv and @ArcLightSlavik ! 👏  🙇 

Sorry @scheung38 , I can't help you if you don't follow the issue template and provide more information.

If you still have problems please create a new issue and follow the template step by step.

Given that, I'm gonna close this one.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
